### PR TITLE
Adding Issue & PR Templates

### LINKED ISSUE #10: Add PR and Issue template

### LINKED ISSUE #11: Update README.md with conventional commit guideline and badge.

### PR DESCRIPTION
**Pull request by vasanthm7**
Fixes #10
Fixes #11 

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added examples for the new scenarios in the relevant modules

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - type: textarea
+    id: exception-or-error
+    attributes:
+      label: Please provide the exception or error you saw
+      render: true
+
+  - type: textarea
+    id: other
+    attributes:
+      label: Anything else?
```
Comment (ankur4u007) [inline .github/ISSUE_TEMPLATE/1-bug-report.yaml]: It's better to end .yaml file with new line.  Same for others yaml in the file

**Review by ankur4u007 — CHANGES_REQUESTED** [review]
LGTM overall. One minor comment. Thanks for taking it up!

**Comment by ankur4u007** [comment]
@vasanthm7 also modify your PR comments from  
`Fixes #10 & #11` 
to
```
Fixes #10 
Fixes #11
 ```
 otherwise, it will not refer the issue. Each individual issues needs to be referenced separately.

File: .github/ISSUE_TEMPLATE/1-bug-report.yaml (hunk 1 above)
Comment (vasanthm7) [reply .github/ISSUE_TEMPLATE/1-bug-report.yaml]: Addressed

**Review by ankur4u007 — APPROVED** [review]

**Pull request merged**